source/user-manual/capabilities/active-response/ar-use-cases/blocking-attacks.rst

```diff
@@ -0,0 +1,177 @@
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+  $ ssh 10.0.0.5
+  Permission denied (publickey).
+
+After 8 attempts, we can see in the manager how the rule is fired:
+
+.. thumbnail:: ../../../../images/manual/automatic-remediation/5712_sshd_brute_force.png
+  :title: Rule 5712 - SSHD brute force trying to get access to the system
+  :align: center
+  :width: 100%
+
+If we try to ping the agent from the attacker, we see that it's not possible:
+
+.. code-block:: none
+
+  [ec2-user@ip-10-0-0-6 ~]$ ping 10.0.0.5
+  PING 10.0.0.5 (10.0.0.5) 56(84) bytes of data.
+  ^C
+  --- 10.0.0.5 ping statistics ---
+  12 packets transmitted, 0 received, 100% packet loss, time 11000ms
+
+
+Active response has blocked the IP so it is working properly.
+
+Generating an alert when an active response is fired
+----------------------------------------------------
+
+Every agent has a log file at ``/var/ossec/logs/active-responses.log`` where the active response activities are registered. By default, this file is being monitored. 
+
+.. code-block:: none
+
+  <ossec_config>
+    <localfile>
+        <log_format>syslog</log_format>
+        <location>/var/ossec/logs/active-responses.log</location>
+    </localfile>
+  </ossec_config>
+
+When the active response is triggered we can see the corresponding alert: 
+
+.. thumbnail:: ../../../../images/manual/automatic-remediation/561_host_blocked_by_firewall_drop.png
+  :title: Rule 561 - host blocked by firewall drop 
+  :align: center
+  :width: 100%
+
+This is possible because rule 651 is defined in ``ossec_rules.xml``. If you create your own script, you must add the proper rule.
+
+White list
+----------
+
+We can also set a list of IP addresses that should never be blocked by the active response. In global section of ``ossec.conf`` in the Manager, use the field ``white_list``. It allows IP address or netblock.
+
+.. code-block:: none
+
+  <ossec_config>
+    <global>
+      <jsonout_output>yes</jsonout_output>
+      <email_notification>no</email_notification>
+      <logall>yes</logall>
+      <white_list>10.0.0.6</white_list>
+    </global>
+
+Increasing blocking time for repeated offenders
+-----------------------------------------------
+
+We set up a blocking time of 30 minutes for our active response, but in case you need to increase this blocking time for repeated offenders you can add the following configuration in the ``ossec.conf`` of each agent:
+
+.. code-block:: none
+
+  <active-response>
+    <repeated_offenders>60,120,180</repeated_offenders>
+  </active-response>
+
+The first time that the active response is triggered, it will block the IP for 30 minutes, the second time for 60 minutes, the third time for 120 minutes and finally the fourth time for 180 minutes.
```

Comment:
```suggestion
The first time that the active response is triggered, it will block the IP for 30 minutes, the second time for 60 minutes, the third time for 120 minutes, and finally the fourth time for 180 minutes.
```